### ISSUE TITLE
Propose improvements in existing integration tests

### ISSUE THREAD

**Issue by sjanota** [kyma-project/kyma#2993]
**Description** 
Currently acceptance tests are not well documented in terms of scope of the test and weight.
We need following information about existing tests gathered:
- Which scenarios are missing yet?
- Which scenarios are no longer valid, relevant?
- How existing tests can be improved?

**Reasons**
We need to create a backlog of missing scenarios to fill the gaps in our test suite. Also if there are any tests that can be removed or improved we need to gather this information to plan the work better in future.

**Acceptance criteria**
For every integration test there is:
- [ ] a list of missing scenarios
- [ ] a list of invalid, redundant, irrelevant scenarios
- [ ] a list of issues with possible improvements for stability/duration

**Comment by aszecowka** [comment]
BTW we can also use that time to check:
- if a test can be run concurrently
- provide consistent labels for Test Definition (to make possible running tests selectively by defining labels)
@sjanota can you add such Acceptance Criteria? 

There is also a similar issue reported by @abbi-gaurav : https://github.com/kyma-project/kyma/issues/3482

**Comment by sjanota** [comment]
@aszecowka What you proposed are very valid points. I included them in previous round of reviews, but it made the scope unmanageable. That's why I focused first on things that needs to be done in cooperation with test owners. Checking if tests can be run concurrently can easily be done in an automated manner. Proposing set of labels may be a side effect of this task, but I don't want to focus on it. We have some labels for now and I think that the rest will naturally come out when we start to use octopus at bigger scale.  

@abbi-gaurav Thanks for the report. Let me close your issue in favour of this one so it is easier to track the progress.
